idlgen/hbs/idl_ex.hbs

```diff
@@ -6,9 +6,9 @@
 {{/each}}
 service {
 {{#each commands.type.def.variant.variants}}
-  {{name}} : ({{#with (lookup @root/all_types fields.[0].type)}}{{#each type.def.composite.fields}}{{#if @index}}, {{/if}}{{name}}: {{{lookup @root/type_names type}}}{{/each}}{{/with}}) -> {{{lookup @root/type_names fields.[1].type}}}
+  command {{name}} : ({{#with (lookup @root/all_types fields.[0].type)}}{{#each type.def.composite.fields}}{{#if @index}}, {{/if}}{{name}}: {{{lookup @root/type_names type}}}{{/each}}{{/with}}) -> {{{lookup @root/type_names fields.[1].type}}};
```

Comment:
I would leave this without prefix. Imo, the semantics would be simpler if methods can mutate state by default unless `query` is specified

Reply:
make sense